### PR TITLE
create model User, Genre and insert to model Track

### PR DESCRIPTION
**Pull request by BeginerDev**
## [Pull request title]
create model user, genres and insert user to track
### [Progress link]
https://docs.google.com/spreadsheets/d/1f-m2BWuOi4DJ5q6SJ2RieOv0f9QGtNeK/edit#gid=581831071

### CODE REVIEW

File: app/src/main/java/com/example/chienmusic/data/model/Genre.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+public class Genre {
+    private String mTitle;
+    private int mImage;
+
+    public Genre() {
+
+    }
+
+    public Genre(String mTitle, int mImage) {
```
Comment (tuannm-1276) [inline app/src/main/java/com/example/chienmusic/data/model/Genre.java]:
```suggestion
    public Genre(String title, int image) {
```sai roi ne

File: app/src/main/java/com/example/chienmusic/data/model/Genre.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        this.mTitle = mTitle;
+        this.mImage = mImage;
+    }
+
+    public String getmTitle() {
```
Comment (tuannm-1276) [inline app/src/main/java/com/example/chienmusic/data/model/Genre.java]: day nua

File: app/src/main/java/com/example/chienmusic/data/model/Genre.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        return mTitle;
+    }
+
+    public void setmTitle(String mTitle) {
```
Comment (tuannm-1276) [inline app/src/main/java/com/example/chienmusic/data/model/Genre.java]: sua lai class nay di

File: app/src/main/java/com/example/chienmusic/data/model/Track.java [hunk 4]
```diff
@@ -36,6 +40,8 @@ public Track(Builder builder) {
         private int mDuration;
         private int mDownloadCount;
         private boolean mIsDownloadable;
+        private User mUser;
+        private String mArtist;
 
         public Builder withArtworkUrl(String artworkUrl) {
```
Comment (tuannm-1276) [inline app/src/main/java/com/example/chienmusic/data/model/Track.java:46]: Co the bo Builder di em.

File: app/src/main/java/com/example/chienmusic/data/model/User.java [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    private String mId;
+    private String mName;
+    private String mAvatarUrl;
+    private String mLinkUser;
+
+    public User() {
+
+    }
+
+    public User(String mId, String mName, String mAvatarUrl, String mLinkUser) {
```
Comment (tuannm-1276) [inline app/src/main/java/com/example/chienmusic/data/model/User.java]: xem laij het nhe

File: app/src/main/java/com/example/chienmusic/data/model/Track.java (hunk 4 above)
Comment (BeginerDev) [reply app/src/main/java/com/example/chienmusic/data/model/Track.java:46]: nhưng nhiều tham số quá cũng bỏ được à anh

**Review by tuannm-1276 — APPROVED** [review]

**Pull request merged**